app/views/notes/index.html.erb

```diff
@@ -1,4 +1,21 @@
 <% notes.each_with_index do |note, i| %>
+
+<div class="row">
+  <div class="card blue-grey file-card col m8 offset-m2">
+      <div class="card-content white-text">
+        <h3 class="card-title">
+          Note <%= i + 1 %>
+        </h3>
+        <div>
+          <div>Name: <%= note.note_with_type %></div>
+          <div>Author: <%= note.user_id %></div>
+          <div>Category: <%= note.category.name %></div>
+          <div>Body: <%= note.body %></div>
+        </div>
+      </div>
+  </div>
+</div>
+
   <% if authorized?(note) %>
```

Comment:
this "<% if authorized?(note) %>" (line 19) should come before the note is shown (line 3 through line 17)